jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -491,7 +491,7 @@ public List<SubscriptionBundle> call() throws Exception {
         final Callable<List<Invoice>> invoicesCallable = new Callable<List<Invoice>>() {
             @Override
             public List<Invoice> call() throws Exception {
-                return invoiceApi.getInvoicesByAccount(accountId, false, false, tenantContext);
+                return invoiceApi.getInvoicesByAccount(accountId, false, false, true, tenantContext); //TODO_1272: should withInvoiceItems be set to false to optimize this call? 
```

Comment:
I believe fetching the invoice items is required: If you follow the code and look at how this `invoicesCallable` is being used, you end up `AccountTimelineJson` where such [items are being used](https://github.com/killbill/killbill/blob/work-for-release-0.23.x/jaxrs/src/main/java/org/killbill/billing/jaxrs/json/AccountTimelineJson.java#L82)